# pressure measurement
float64 pre_FB1
float64 pre_FB2
float64 pre_FB3
float64 pre_FB4
# cable measurement
float64 len_FB1
float64 len_FB2
float64 len_FB3
float64 len_FB4








